### DISCUSSION TITLE
Provide custom ThreadFactory

### DISCUSSION THREAD

**Discussion by KBA2024** [kagkarlsson/db-scheduler#586] [Q&A] [answered]
How can I customize the thread information in db-scheduler? 
Currently, it uses PrefixingDefaultThreadFactory, which relies on java.util.concurrent.Executors.defaultThreadFactory(). In my case, I need the threads that execute tasks to include custom information. What would be the best way to do it?

**Comment by kagkarlsson** [comment, marked as answer]
You can override the `ExecutorService` used in the `SchedulerBuilder` or via `DbSchedulerCustomizer` (for spring boot)
